src/app/germplasm/germplasm-pedigree-graph-modal.component.ts

```diff
@@ -63,7 +63,7 @@ export class GermplasmPedigreeGraphModalComponent implements OnInit {
     });
 
     let pedigreeMapDestination: Map<string, PedigreeNode> = new Map<string, PedigreeNode>();
```

Comment:
@nahuel-soldevilla We will need to adjust the v1.8.0 tag in brapi-sync once this is merged.